### ISSUE TITLE
Build failure with servant 0.16

### ISSUE THREAD

**Issue by snoyberg** [tonatona-project/tonatona#40]
```
[2 of 2] Compiling Tonatona.Servant ( src/Tonatona/Servant.hs, dist/build/Tonatona/Servant.o )

src/Tonatona/Servant.hs:53:53: error:
    Not in scope: type constructor or class ‘ServantErr’
   |
53 |       eitherRes <- liftIO $ ioAction `catch` \(e :: ServantErr) -> pure $ Left e
   |                                                     ^^^^^^^^^^
```

**Comment by arowM** [comment]
Thanks. I've released tonatona-servant-0.1.0.2 that supports servant 0.16.

**Comment by snoyberg** [comment]
Great, thanks!
